web-client/iron-remote-gui/README.md

```diff
@@ -52,12 +52,31 @@ You can add some parameters for default initialization on the component `<iron-r
 
 ## `UserInteractionService` methods
 
-> `connect(username: string, password: string, host: string, authtoken: string): Observable<NewSessionInfo>`
+```typesccript
+ private connect(
```

Comment:
I think you should copy from `UserInteraction.ts` instead:
```ts
    connect(
        username: string,
        password: string,
        destination: string,
        proxyAddress: string,
        serverDomain: string,
        authToken: string,
        desktopSize?: DesktopSize,
        preConnectionBlob?: string,
        kdc_proxy_url?: string,
    ): Observable<NewSessionInfo>;
```

(no `private`)

Reply:
updated